LabSid-USP/RUBEM
# Enable export of the resulting Total Runoff (RNF) raster series and time series

### Checklist

### Description


- It continues the work started in #130;
- Added Total Runoff (RNF) (`self.Qtot`) as raster series output;
https://github.com/LabSid-USP/RUBEM/blob/271e074a768dd6cc843c826c19b9c78791a453ac/rubem/_dynamic_model.py#L598

### Related Issue
  
 

- Resolve #124.

### Motivation and context


- See #124.

### How has this been tested


- The behavior and simulation results of the model must remain unchanged in relation to the current implementation:

    1. Set up a simulation enabling all simulation outputs (rasters and tables), specifically "Total Runoff (RNF)" and "Accumulated Total Runoff (ARN)";
    2. Run the simulation;
    3. Verify that the specified rasters and tables files were created in the output directory.

### Screenshots


- N/A
